process_report/process_report.py

```diff
@@ -333,6 +364,23 @@ def validate_pi_names(dataframe):
     return dataframe
 
 
+def validate_pi_aliases(dataframe: pandas.DataFrame, alias_dict: dict):
+    for pi, pi_aliases in alias_dict.items():
+        for pi_alias in pi_aliases:
+            if not dataframe.loc[dataframe[PI_FIELD] == pi_alias].empty:
+                print(f"Warning: Found alias {pi_alias} for {pi}")
+            dataframe.loc[dataframe[PI_FIELD] == pi_alias, PI_FIELD] = pi
+
+    return dataframe
+
+
+def fetch_s3_alias_file():
+    local_name = "alias.csv"
+    invoice_bucket = get_invoice_bucket()
+    invoice_bucket.download_file(ALIAS_S3_FILEPATH, local_name)
+    return local_name
```

Comment:
It's a little weird that we're returning a static filename to the caller. Should this just be a global constant instead?

Reply:
I only use it once in this function, so I didn't thought it would be necessary to make it a global. Should I?

Reply:
I guess my question is why we're bothing to return a value here at all, since it's always the same thing.

Reply:
I...um...don't have a strong reason why it's like that? I guess I just wanted all the related code in one place, but if you think it's better to make this a constant global, I'll do so.

Reply:
I don't have a strong opinion here. It's fine if you leave it the way it is for now; we can always change it later if we develop a compelling reason to do so.